Scripts/RemediationScripts/Remediate-EnableEncryptionInTransitForStorageAccounts.ps1

```diff
@@ -0,0 +1,686 @@
+﻿<##########################################
+
+# Overview:
+    This script is used to make the transit in Storage Account encrypted using the HTTPS.
+
+# ControlId: 
+    Azure_Storage_DP_Encrypt_In_Transit
+
+# DisplayName:
+    Enable Secure transfer to Storage Accounts
+
+# Pre-requisites:
+    You will need atleast contributor role on Storage Account(s) of subscription.
+
+# Steps performed by the script :
+   To Remediate:
+        1. Checking for prerequisites..
+        2. Validating the Account type.
+        3. Removing the excluded resource group(s) and resource from the list of subscription.
+        4. Get list of Storage Account in a subscription which are having disable secure transfer.
+        5. Exporting the Storage Account(s) to csv file so that it can be used to rollback.
+        6. Enable secure transfer for the Storage Account(s) of subscription.
+
+   To rollback:
+        1. Checking for Prerequisites..
+        2. Validating the Account type.
+        3. Get the list of Storage Accounts in a Subscription, the changes made to which previously, are to be rolled back.
+        4. Disable secure transfer on the Storage Account in the Subscription.
+
+# Instructions to execute the script:
+    To remediate:
+        1. Download the script.
+        2. Load the script in a PowerShell session. Refer https://aka.ms/AzTS-docs/RemediationscriptExcSteps to know more about loading the script.
+        3. Load the script in the file Helper.ps1 available in the Remediation Script directory.
+        4. Execute the script To enable encrpytion in transit on the Storage Account in the Subscription. Refer 'Examples', below.
+   
+    To roll back:
+        1. Download the script.
+        2. Load the script in a PowerShell session. Refer https://aka.ms/AzTS-docs/RemediationscriptExcSteps to know more about loading the script.
+        3. Execute the script to disable  encrpytion in transit" on the Storage Account in the Subscription. Refer `Examples`, below.
+
+# Examples:
+    To remediate:
+        1. To review the Storage Account in a Subscription that will be remediated:
+           Enable-StorageEncryptionInTransit -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 -DryRun
+
+        2. To enable encrpytion in transit on the Storage Account in a Subscription:
+           Enable-StorageEncryptionInTransit -SubscriptionId 00000000-xxxx-0000-xxxx-000000000000 
+       
+        3. To enable  encrpytion in transit on the Storage Account in a Subscription, from a previously taken snapshot:
```

Comment:
Typo: `encrpytion` to be replaced with `encryption`.

Reply:
done